CMakeModules/CPackConfig.cmake

```diff
@@ -10,10 +10,10 @@ cmake_minimum_required(VERSION 3.5)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/CMakeModules/nsis")
 
 include(Version)
-include(CPackIFW)
+
+set(CPACK_THREADS 16)
```

Comment:
16 is too high and can cause hangs if any users attempt to build it themselves.

Reply:
Its not really used from my tests.

Reply:
can't we make this 8 ? most modern laptops will be fine with that. there is no need to make experience bad if and when someone takes a stab at it.